scss/_progress.scss

```diff
@@ -7,101 +7,69 @@
   to    { background-position: 0 0; }
 }
 
-
 //
 // Basic progress bar
 //
 
 .progress {
+  @include border-radius($progress-border-radius);
+  @include box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
+  background-color: $progress-bg;
```

Comment:
Properties should be ordered display, width, height, margin-bottom, background-color